src/spec.md

```diff
@@ -0,0 +1,190 @@
+
+- `update.max_serial`: the maximum serial currently known.
+  If `max_serial` equals `serial` this update is the last update (until new network messages arrive).
+
+- `update.info`: optional, short, informational message (see `sendUpdate()`)
+
+- `update.document`: optional, document name as set by the sender, (see `sendUpdate()`),
+  implementations show the document name eg. beside the app icon or in the title bar
+
+- `update.summary`: optional, short text, shown beside icon (see `sendUpdate()`)
+
+
+### selfAddr
+
+```js
+window.webxdc.selfAddr
+```
+
+Property with the peer's own address.
+This is esp. useful if you want to differ between different peers -
+just send the address along with the payload,
+and, if needed, compare the payload addresses against selfAddr() later on.
+
+
+### selfName
+
+```js
+window.webxdc.selfName
+```
+
+Property with the peer's own name.
+This is name chosen by the user in their settings,
+if there is nothing set, that defaults to the peer's address.
+
+
+
+## Other APIs and Tags Usage Hints
+
+- `localStorage`, `sessionStorage`, `indexedDB` are okay to be used
+- `visibilitychange`-events are okay to be used
+- `window.navigator.language` is okay to be used, on desktop it is the system language
+- `<a href="localfile.html">` and other internal links are okay to be used
+- `<a href="mailto:addr@example.org?body=...">`- mailto links are okay to be used
+- `<meta name="viewport" ...>` usage is okay to be used
+  and useful esp. different webviews have different defaults
+
+
+### Discouraged practises 
+
+- `document.cookie` is known not to work on desktop and iOS
+  use `localStorage` instead
+- `unload`-, `beforeunload`- and `pagehide`-events are known not to work on iOS and are flaky on other systems
+  (also partly discouraged by [mozilla](https://developer.mozilla.org/en-US/docs/Web/API/Window/unload_event))
+  use `visibilitychange` instead
+- `<title>` and `document.title` is ignored by Webxdc;
+  use the `name` property from `manifest.toml` instead
+- newest js features may not work on all webviews,
+  you may want to transpile your code down to an older js version
+  eg. with <https://babeljs.io>
+- `<a href="https://example.org/foo">` and other external links are blocked by definition;
+  instead, embed content or use `mailto:` link to offer a way for contact
+- `<input type="file">` is discouraged currently; this may change in future
+
+
+## Webxdc File Format
+
+- a **Webxdc** is a **ZIP-file** with the extension `.xdc`
+- the ZIP-file MUST use the default compression methods as of RFC 1950,
+  this is "Deflate" or "Store"
+- the ZIP-file MUST contain at least the file `index.html`
+- the ZIP-file MAY contain a `manifest.toml` and `icon.png` or
+  `icon.jpg` files
+- if the Webxdc app is started, `index.html` MUST be opened in a [restricted
+  webview](Webview constraints for running apps) that allow accessing 
+  resources only from the ZIP-file.
+
+### The manifest.toml file
```

Comment:
```suggestion
### The manifest.toml File
```

Reference: https://www.grammarunderground.com/capitalization-of-headlines.html
(I would keep technical terms like file names in their original capitalization though)